### PR TITLE
Feature/101/login page remade

### PR DESCRIPTION
**Pull request by bytewiz**
#130 Fixes from this pull request

### CODE REVIEW

File: src/client/pages/Login/LoginPage.js [hunk 1]
```diff
@@ -1,7 +1,130 @@
...
+  //     if (user) {
+  //       this.setState({ user });
+  //       localStorage.setItem('user', user.uid);
+  //     } else {
+  //       this.setState({ user: null });
+  //       localStorage.removeItem('user');
+  //     }
+  //     this.setState({ isLoading: false });
+  //   });
+  // }
```
Comment (JonathanMH) [inline src/client/pages/Login/LoginPage.js]: Can we remove the comments or are they required to make the authentication work and connect our `user.id` field to firebase?

**Review by JonathanMH — COMMENTED** [review]
Just curious about the comments, otherwise I assume the social auth works

File: src/client/pages/Login/LoginPage.js (hunk 1 above)
Comment (berheg) [reply src/client/pages/Login/LoginPage.js]: Yes, agree to remove comments otherwise if you want import auth do not forget to setState isLoading state.

**Review by OlkaA — APPROVED** [review]
Authentication works.
Commented code need to be deleted.

**Pull request merged**